trueos-handbook/introducing.rst

```diff
@@ -475,83 +478,129 @@ manager utilities.
    :class: longtable
 
    +------------+-----------+--------------+--------------------------------------------------------+
-   | Filesystem | Native to | Non-native   | Usage notes                                            |
-   |            |           | support type |                                                        |
+   | Filesystem | Native OS | Non-native OS| Usage notes                                            |
+   |            |           | support      |                                                        |
    +============+===========+==============+========================================================+
-   | Btrfs      | Linux     | none         |                                                        |
+   | Btrfs      | Linux     | none         | A modern copy on write (CoW) filesystem for the Linux  |
+   |            |           |              | OS. Btrfs is similar in nature to ZFS, and shares many |
+   |            |           |              | of the same ideas with how a file system should work.  |
+   |            |           |              | The maximum supported volume and file size is 16EB.    |
```

Comment:
Done. This is going to be an issue for all of the filesystem entries. I looked all of them up to find the max supported volume and file size. As Btrfs is in active development, the max can change in the future, but entries for say fat16 probably not so much. I'll leave the entries as is until there is more discussion. For this entry specifically, and tables in rst in general, does rst automagically handle the borders s long as there is an ending pipe? The link is super long and didn't want to line break the URL. It would make sense that this is handled by rst, but wanted to ask.